rainier-core/src/main/scala/com/stripe/rainier/core/Predictor.scala

```diff
@@ -5,66 +5,65 @@ import com.stripe.rainier.compute._
 /**
   * Predictor class, for fitting data with covariates
   */
-trait Predictor[X, Y] extends Likelihood[(X, Y)] {
-  private[core] type Q
-  private[core] type R
-  type P = (Q, R)
-  private[core] def create(q: Q): Distribution.Aux[Y, R]
-  private[core] def xq: Wrapping[X, Q]
+sealed trait Predictor[L, X] { self =>
```

Comment:
This is a huge nit, but since this contains a type `P`, an `Encoder[X, P]` and a function `P => L`, it would be less confusing for it to be a `Predictor[X, L]`.

Reply:
Yep will do.